tensorflow-examples/src/main/java/org/tensorflow/model/examples/cnn/fastrcnn/FasterRcnnInception.java

```diff
@@ -101,26 +101,25 @@ The given SavedModel SignatureDef contains the following output(s):
 */
 
 
-import org.tensorflow.Graph;
-import org.tensorflow.SavedModelBundle;
-import org.tensorflow.Session;
-import org.tensorflow.Tensor;
+import org.tensorflow.*;
```

Comment:
We prefer that there are no star imports, so you can see all the classes that are used.

Reply:
That's IntelliJ Idea, seems it has a 'helpful' feature that occasionally decides to merge those classes from the same package.

Reply:
Yeah, you can turn the threshold up at which it decides to do that. I have it set to 999.

Reply:
Found that thanks